dev/com.ibm.ws.channelfw/src/com/ibm/ws/tcpchannel/internal/WorkQueueManager.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2005, 2021 IBM Corporation and others.
+ * Copyright (c) 2005, 2025 IBM Corporation and others.
```

Comment:
@mrsaldana can you just remove the contribors from the copyright header since it is no longer necessary? It is good practice to clean these up as we touch files. Thanks!